src/components/record/related-table-actions.tsx

```diff
@@ -28,48 +34,73 @@ import { ConfigService } from '@isrd-isi-edu/chaise/src/services/config';
 
 // utils
 import { addQueryParamsToURL } from '@isrd-isi-edu/chaise/src/utils/uri-utils';
-import { allowCustomModeRelated, displayCustomModeRelated, getPrefillCookieObject } from '@isrd-isi-edu/chaise/src/utils/record-utils';
-import { RECORDSET_DEAFULT_PAGE_SIZE, CUSTOM_EVENTS } from '@isrd-isi-edu/chaise/src/utils/constants';
+import {
+  allowCustomModeRelated,
+  displayCustomModeRelated,
+  getPrefillCookieObject,
+} from '@isrd-isi-edu/chaise/src/utils/record-utils';
+import {
+  RECORDSET_DEAFULT_PAGE_SIZE,
+  CUSTOM_EVENTS,
+} from '@isrd-isi-edu/chaise/src/utils/constants';
 import { windowRef } from '@isrd-isi-edu/chaise/src/utils/window-ref';
 import { getRandomInt } from '@isrd-isi-edu/chaise/src/utils/math-utils';
 import { fireCustomEvent } from '@isrd-isi-edu/chaise/src/utils/ui-utils';
 import Q from 'q';
+import ResizeSensor from 'css-element-queries/src/ResizeSensor';
 
 type RelatedTableActionsProps = {
-  relatedModel: RecordRelatedModel
-}
+  relatedModel: RecordRelatedModel;
+  istableHeader?: boolean;
```

Comment:
You don't need this new prop. Just use `relatedModel.isInline` to check if this is used in the main section or the related section.